app/models/entity.rb

```diff
@@ -181,6 +181,7 @@ class Entity < Ekylibre::Record::Base
   scope :normal, -> { where(of_company: false) }
   scope :necessary_transporters, -> { where("transporter OR id IN (SELECT transporter_id FROM #{Parcel.table_name} WHERE state != 'sent' OR delivery_id IS NULL)").order(:last_name, :first_name) }
   scope :suppliers,    -> { where(supplier: true) }
+  scope :of_company,   -> { where(of_company: true) }
```

Comment:
Please use instead class method define below which return the unique Entity record and not a relation